js/app.js

```diff
@@ -20,231 +20,254 @@ allAnswers = [];
 realAnswers = [];
 countries = ['canada', 'mexico', 'portugal'];
 
-//start questions
-alert('Hey, welcome to my site. My name is Austin Matteson but you may already know that. In fact, let\'s find out just how much you do know about me.');
-userName = prompt('But first, what is your name?');
-console.log('The user entered', userName);
-alert('Great name, ' + userName + ', let\'s begin the test!');
+function person() {
```

Comment:
Just so you know, generally functions are named with verbs to help indicate that it does something. I.e. instead of naming this function `person` you could name it something like `getPersonName`.